blocks/gallery-block/features/gallery/default.jsx

```diff
@@ -102,8 +101,8 @@ export const GalleryPresentation = ({
 				</button>
 			}
 			fullScreenMinimizeButton={
-				<button type="button">
-					<Icon name="Close" />
+				<button type="button" className={`${BLOCK_CLASS_NAME}__button--exit-full-screen`}>
```

Comment:
This should be the `Button` component from `arc-themes-components`.

Reply:
Comment attended on refactoring.